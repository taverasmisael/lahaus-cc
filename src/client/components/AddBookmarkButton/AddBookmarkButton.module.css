.container {
  @apply overflow-hidden grid items-center;
}
.header {
  @apply bg-secondary bg-opacity-25 relative block text-white flex-1 w-full rounded-lg;

  height: 196px;
  max-height: 196px;
}
.icon {
  @apply absolute top-1/2 left-1/2 transform -translate-y-1/2 -translate-x-1/2;
}
.copy {
  @apply mb-2 mt-4 text-secondary font-semibold;
}
