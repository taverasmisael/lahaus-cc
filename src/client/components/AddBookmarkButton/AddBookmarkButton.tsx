import PlusIcon from '../../assets/plus-icon.svg'
import styles from './AddBookmarkButton.module.css'

function AddBookmarkButton() {
  return (
    <button
      type="button"
      className={styles.container}
      onClick={() => alert(`I'm not implementing this either ðŸ˜¬`)}
    >
      <span className={styles.header}>
        <PlusIcon className={styles.icon} />
      </span>
      <span className={styles.copy}>Crear una nueva lista</span>
    </button>
  )
}

export default AddBookmarkButton
