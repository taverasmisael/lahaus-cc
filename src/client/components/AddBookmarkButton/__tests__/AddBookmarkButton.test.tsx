import { render } from '@testing-library/react'

import AddBookmarkButton from '../AddBookmarkButton'

describe('components/AddBookmarkButton', () => {
  it('renders correctly', () => {
    const { container } = render(<AddBookmarkButton />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
