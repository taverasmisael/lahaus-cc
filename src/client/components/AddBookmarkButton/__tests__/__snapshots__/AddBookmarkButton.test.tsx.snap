// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/AddBookmarkButton renders correctly 1`] = `
<button
  class="container"
  type="button"
>
  <span
    class="header"
  >
    <svg
      class="icon"
      fill="none"
      height="38"
      width="38"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M20.875 17.125V.25h-3.75v16.875H.25v3.75h16.875V37.75h3.75V20.875H37.75v-3.75H20.875Z"
        fill="#3948FF"
      />
    </svg>
  </span>
  <span
    class="copy"
  >
    Crear una nueva lista
  </span>
</button>
`;
